vm/asm.go

```diff
@@ -16,15 +16,45 @@
 
 package vm
 
+import (
+	"github.com/DE-labtory/koa/opcode"
+	"github.com/pkg/errors"
+)
+
+var opCodes = map[opcode.Type]opCode{
+	opcode.Add:  add{},
+	opcode.Push: push{},
+}
+
 // Converts rawByteCode to assembly code.
 func assemble(rawByteCode []byte) (*asm, error) {
-	analysis()
-	return &asm{}, nil
+	asm := newAsm()
+
+	for i := 0; i < len(rawByteCode); i++ {
+		op, ok := opCodes[opcode.Type(rawByteCode[i])]
+
+		if !ok {
+			return nil, errors.New("Invalid byteCode")
+		}
```

Comment:
I think it's better to predefine 
`var ErrInvalidOpcode = errors.New("Invalid byteCode")` and return `ErrInvalidOpcode`